elide-model-config/src/test/java/com/yahoo/elide/modelconfig/validator/DynamicConfigValidatorTest.java

```diff
@@ -364,6 +365,66 @@ public void testJoinedTablesDBConnectionNameMismatch() throws Exception {
         assertTrue(error.contains(" and tables in its Join Clause."));
     }
 
+    @Test
+    public void testArgumentMissingForSqlInvokedColumn() throws Exception {
+        String error = tapSystemErr(() -> {
+            int exitStatus = catchSystemExit(() -> DynamicConfigValidator
+                    .main(new String[]{"--configDir", "src/test/resources/validator/sqlhelper_missing_arguments"}));
+
+            assertEquals(2, exitStatus);
+        });
+        assertEquals("Can't invoke 'revenue' provided in \"sql from='revenueFact' column='revenue[currency:USD]'\" for Column: 'impressionsPerUSD' in Model: 'revenueFact'. Argument: 'format' with type: 'TEXT' is required for column: 'impressionsPerUSD'\n",
+                        error);
+    }
+
+    @Test
+    public void testSqlInvokedModelMissing() throws Exception {
+        String error = tapSystemErr(() -> {
+            int exitStatus = catchSystemExit(() -> DynamicConfigValidator
+                    .main(new String[]{"--configDir", "src/test/resources/validator/sqlhelper_invalid_invoked_model"}));
+
+            assertEquals(2, exitStatus);
+        });
+        assertEquals("Can't invoke 'revenueFactInvalid' provided in \"sql from='revenueFactInvalid' column='impressions[aggregation:SUM]'\". 'From' must match with either current table name or any of the Join names\n",
+                        error);
+    }
+
+    @Test
+    public void testMissingReferredColumn() throws Exception {
+        String error = tapSystemErr(() -> {
+            int exitStatus = catchSystemExit(() -> DynamicConfigValidator
+                    .main(new String[]{"--configDir", "src/test/resources/validator/invalid_referred_column"}));
+
+            assertEquals(2, exitStatus);
+        });
+        assertEquals("Can't invoke 'revenue' provided in \"sql from='revenueFact' column='revenue[currency:USD]'\" for Column: 'impressionsPerUSD' in Model: 'revenueFact'. Column: 'revenue' is undefined for hjson model: 'revenueFact'\n",
```

Comment:
Models may not come from HJSON.  The returned text should not reference HJSON here.